tests/test_client.py

```diff
@@ -550,5 +553,41 @@ def test_response_is_http_error(self):
             self.assertEqual(e.exception.reason, api_res.reason)
             self.assertEqual(e.exception.data, res_body)
 
+
+class TestSSLConfig(unittest.TestCase):
+    """ Tests the ability to specify override our default ssl context and use sslv3 """
+    def argument_used(self, call_args_list, expected_arg):
+        """ Assert that an argument is used at least once in any of the calls to a function"""
+        for call in call_args_list:
+            args, kwargs = call
+            if expected_arg in args:
+                return True
+        return False
+
+    @mock.patch('duo_client.https_wrapper.CertValidatingHTTPSConnection.add_option_to_ssl_context')
+    def test_safe_default_https_wrapper(self, mock_add_option):
+        """ sslv2 and sslv3 are off """
+        mock_add_option.return_value = ssl.SSLContext(ssl.PROTOCOL_SSLv23)
+        CertValidatingHTTPSConnection('fake host')
+
+        arg_used = self.argument_used(mock_add_option.call_args_list, ssl.OP_NO_SSLv2)
```

Comment:
I think using `mock_add_option.assert_called_with(ssl.OP_NO_SSLv2)` would be cleaner and eliminate the need for the above method.